dispatch/dispatch.go

```diff
@@ -378,14 +378,17 @@ func (d *Dispatcher) processAlert(dispatchLink trace.Link, alert *types.Alert, r
 		)
 		defer span.End()
 
-		_, _, err := d.stage.Exec(ctx, d.logger, alerts...)
+		pipelineTime, _ := notify.Now(ctx)
```

Comment:
If you want to be accurate with the name, it's really a flush event:

```suggestion
		flushTime, _ := notify.Now(ctx)
```

Reply:
Yeah, that's a good point. I'll leave `pipelineTime` for now though as this started being used in other places, but I'll raise this in the next opportunity in case we'd like to rename it 👍